azure/packages/test/end-to-end-tests/src/test/audience.spec.ts

```diff
@@ -161,4 +156,172 @@ describe("Fluid audience", () => {
 		members = servicesGet.audience.getMembers();
 		assert.strictEqual(members.size, 1, "We should have one member left at this point.");
 	});
+
+	/**
+	 * Scenario: Find read-only partner member
+	 *
+	 * Expected behavior: upon resolving container, the read-only partner member should be able
+	 * to resolve original member, and the original member should be able to observe the read-only member.
+	 */
+	it("can find read-only partner member", async function () {
+		if (process.env.FLUID_CLIENT !== "azure") {
+			// Tinylicious does not support read-only mode
+			this.skip();
+		}
+		const { container, services } = await client.createContainer(schema);
+		const containerId = await container.attach();
+
+		if (container.connectionState !== ConnectionState.Connected) {
+			await timeoutPromise((resolve) => container.once("connected", () => resolve()), {
+				durationMs: connectTimeoutMs,
+				errorMsg: "container connect() timeout",
+			});
+		}
+
+		assert.strictEqual(typeof containerId, "string", "Attach did not return a string ID");
+		assert.strictEqual(
+			container.attachState,
+			AttachState.Attached,
+			"Container is not attached after attach is called",
+		);
+
+		/* This is a workaround for a known bug, we should have one member (self) upon container connection */
```

Comment:
Would be nice to have a TODO comment here (and the other instances of this comment) with a link to the ADO item for the bug

Reply:
Do you happen to have the link for the bug? I just copied this comment from elsewhere

Reply:
Ah I see, I didn't realize it was being copied from other tests. Unless we find the bug we can just leave it